diff --git a/libs/content.ts b/libs/content.ts
index 9e2c7fc619457e6ac32af953896985fd4d4dfb24..e9331ee37675dcc8c6adf79476209a2bf510bdd9 100644
--- a/libs/content.ts
+++ b/libs/content.ts
@@ -4 +3,0 @@
-import context from 'virtual:starlight-blog/context'
@@ -92,2 +90,0 @@
-  const contentRelativePath = `${context.srcDir.replace(context.rootDir, '')}content/docs/`
-
@@ -97,2 +93,0 @@
-    const fileRelativePath = entry.filePath?.replace(contentRelativePath, '')
-
@@ -100,2 +95,2 @@
-      fileRelativePath?.startsWith(`${getPathWithLocale(config.prefix, DefaultLocale)}/`) &&
-      fileRelativePath !== `${getPathWithLocale(config.prefix, DefaultLocale)}/index.mdx`
+      entry.id.startsWith(`${getPathWithLocale(config.prefix, DefaultLocale)}/`) &&
+      entry.id !== `${getPathWithLocale(config.prefix, DefaultLocale)}/index`
